services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -380,7 +380,7 @@ private void onStoreInitialized() {
 
   @Override
   public void onTick(Date date) {
-    if (chainStorageClient.isPreGenesis() || syncService.isSyncActive()) {
+    if (chainStorageClient.isPreGenesis()) {
```

Comment:
The simplest solution is that we revert commit 66e18fa for now and come up with a proper solution for #1497 - this would leave all the logging implemented but still aborts early (therefore not posting synclogs - _yet_).

Otherwise, 18c0ed2 looks good to me if you also prefer to have some sync logging.